test/integration/integration-test.js

```diff
@@ -188,81 +168,73 @@ describe.skip('integration tests', function() {
   });
 
 
-  it('getServerInfo', function(done) {
-    this.api.getServerInfo(composeAsync(data => {
-      assert(data && data.info && data.info.pubkey_node);
-    }, done));
+  it('getServerInfo', function() {
+    return this.api.getServerInfo().then(data => {
+      assert(data && data.pubkeyNode);
+    });
   });
 
 
   it('getFee', function() {
     const fee = this.api.getFee();
     assert.strictEqual(typeof fee, 'string');
-    assert(!isNaN(+fee));
+    assert(!isNaN(Number(fee)));
```

Comment:
Should probably assert(parseInt(fee) === Number(fee)) or assert(/^\d+$/.test(fee))